astroplan/core.py

```diff
@@ -0,0 +1,234 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..extern import six
+
+import sys
+from math import sqrt, pi, exp, log, floor
+from abc import ABCMeta, abstractmethod
+
+import numpy as np
+
+# Originally authored by Jazmin Berlanga Medina (jazmin.berlanga@gmail.com),
+# and modified by Adrian Price-Whelan (email) and Erik Tollerud (email).
+
+__all__ = ["Observer", "Date", "Target", "FixedTarget", "NonFixedTarget",
+           "Constraints", "Observation"]
+
+#__doctest_requires__ = {'*': ['scipy.integrate']}
+
+
+class Observer(object):
+    """
+    Some comments.
+    """
+    #def __init__(self, name, longitude, latitude, elevation, pressure,
```

Comment:
@adrn @eteq I commented out several things because I wasn't sure exactly how to implement each function.  

I'll go back & uncomment/etc. & see if it runs without errors.